notify/victorops/victorops.go

```diff
@@ -70,7 +73,19 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 		tmpl   = notify.TmplText(n.tmpl, data, &err)
 		apiURL = n.conf.APIURL.Copy()
 	)
-	apiURL.Path += fmt.Sprintf("%s/%s", n.conf.APIKey, tmpl(n.conf.RoutingKey))
+
+	var apiKey string
+	if n.conf.APIKey != "" {
+		apiKey = string(n.conf.APIKey)
+	} else {
+		content, fileErr := os.ReadFile(n.conf.APIKeyFile)
```

Comment:
I might miss the reason why we need a new variable name but if there's none, I'd prefer to keep `err` :) 
```suggestion
		content, err := os.ReadFile(n.conf.APIKeyFile)
```

Reply:
Declaring it as `err` would be a shadow declaration of a previously defined `err` variable used for templating errors: 🙂 
https://github.com/prometheus/alertmanager/blob/3f561cdce69ecff5cb7cff1a611b3fc592e964a9/notify/victorops/victorops.go#L70-L75

Reply:
That makes sense! I agree that the templating error handling is not the best, just need to be careful I guess :)

Reply:
Are there any other changes needed for this PR btw? 🏁